test/integration/integration-test.js

```diff
@@ -74,17 +78,110 @@ function setup(server = 'wss://s1.ripple.com') {
   });
 }
 
+const masterAccount = 'rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh';
+const masterSecret = 'snoPBrXtMeMyMHUVTgbuqAfg1SUTb';
+
+function makeTrustLine(testcase, address, secret) {
+  const api = testcase.api;
+  const specification = {
+    currency: 'USD',
+    counterparty: masterAccount,
+    limit: '1341.1',
+    ripplingDisabled: true
+  };
+  const trust = api.prepareTrustline(address, specification, {})
+    .then(data => api.sign(data.txJSON, secret))
```

Comment:
`sign` is synchronous, so you don't have to put it in a `then`, it can go into the next block